ui/src/pages/Results_test.js

```diff
@@ -0,0 +1,115 @@
+import Enzyme from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import toJson from 'enzyme-to-json';
+import {createMemoryHistory} from 'history';
+import React from 'react';
+import {MemoryRouter} from 'react-router';
+
+import Results from './Results';
+import RepoHeader from '../components/RepoHeader';
+import SearchBar from '../components/SearchBar';
+import {mountWithIntl} from '../testing/enzyme-intl-helper';
+import {flushPromises} from '../testing/utils';
+
+Enzyme.configure({adapter: new Adapter()});
+
+const REPO_DATA = {repoId: 'albany', title: 'Albany', recordCount: 100,};
```

Comment:
Is 100 really the population of Albany?

Reply:
:p No, although the population is gradually declining, so, you know, eventually...